src/index.ts

```diff
@@ -79,7 +79,7 @@ export function getFormattedSrcLink(filepath: string, repoUrl?: RepoUrl) {
 export default async function todos({
   repoUrl = true,
   ignore = [],
-  keywords = ["TODO", "FIXME"],
+  keywords = ["TODO", "FIXME", "BUGBUG"],
```

Comment:
Just one nitpick, is 'BUGBUG' a common pattern? It's the first time I'm seeing it. So if it's something that you personally use than I'd prefer not adding it to defaults.

Reply:
```suggestion
  keywords = ["TODO", "FIXME"],
```